Account | test_rename_account | Traceback (most recent call last):
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/accounts/doctype/account/test_account.py", line 64, in test_rename_account
    self.assertEqual(account_number, "1210")
AssertionError: '1211-11-4 - 6 -' != '1210'
- 1211-11-4 - 6 -
+ 1210

### Issue Details:
Traceback (most recent call last):
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/accounts/doctype/account/test_account.py", line 64, in test_rename_account
    self.assertEqual(account_number, "1210")
AssertionError: '1211-11-4 - 6 -' != '1210'
- 1211-11-4 - 6 -
+ 1210

### Traceback:
``` 

ERROR: test_rename_account (erpnext.accounts.doctype.account.test_account.TestAccount)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/accounts/doctype/account/test_account.py", line 70, in test_rename_account
    update_account_number("1210 - Debtors - _TC", new_account_name, new_account_number)
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/utils/typing_validations.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/accounts/doctype/account/account.py", line 511, in update_account_number
    frappe.rename_doc("Account", name, new_name, force=1)
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/utils/typing_validations.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/__init__.py", line 1452, in rename_doc
    return rename_doc(
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/model/rename_doc.py", line 164, in rename_doc
    new = validate_rename(
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/model/rename_doc.py", line 376, in validate_rename
    frappe.throw(_("Another {0} with name {1} exists, select another name").format(doctype, new))
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/__init__.py", line 632, in throw
    msgprint(
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/__init__.py", line 597, in msgprint
    _raise_exception()
  File "/home/frappe/test-postgres/test-bench/apps/frappe/frappe/__init__.py", line 548, in _raise_exception
    raise exc
frappe.exceptions.ValidationError: Another Account with name 1211-11-4 - 6 - - Debtors 1 - Test - - _TC exists, select another name)))
```

--- a/erpnext/accounts/doctype/account/test_account.py
+++ b/erpnext/accounts/doctype/account/test_account.py
@@ -50,6 +50,9 @@ class TestAccount(unittest.TestCase):
 		frappe.delete_doc("Account", "Child Account 2 - _TC")
 
 	def test_rename_account(self):
+		if frappe.db.exists("Account", "1211-11-4 - 6 - - Debtors 1 - Test - - _TC"):
+			frappe.delete_doc("Account", "1211-11-4 - 6 - - Debtors 1 - Test - - _TC")
+
 		if not frappe.db.exists("Account", "1210 - Debtors - _TC"):
 			acc = frappe.new_doc("Account")
 			acc.account_name = "Debtors"
@@ -58,11 +61,11 @@ class TestAccount(unittest.TestCase):
 			acc.company = "_Test Company"
 			acc.insert()
 
-		account_number, account_name = frappe.db.get_value(
-			"Account", "1210 - Debtors - _TC", ["account_number", "account_name"]
-		)
-		self.assertEqual(account_number, "1210")
-		self.assertEqual(account_name, "Debtors")
+			account_number, account_name = frappe.db.get_value(
+				"Account", "1210 - Debtors - _TC", ["account_number", "account_name"]
+			)
+			self.assertEqual(account_number, "1210")
+			self.assertEqual(account_name, "Debtors")
 
 		new_account_number = "1211-11-4 - 6 - "
 		new_account_name = "Debtors 1 - Test - "

